.storybook/storybook-data.js

```diff
@@ -167,6 +167,20 @@ module.exports = [
 			},
 		},
 	},
+	{
+		id: 'global--modules-list',
+		kind: 'Global',
+		name: 'Modules List',
+		story: 'Modules List',
```

Comment:
This issue doesn't call for a new VRT scenario to be added since this component is already covered by other existing scenarios, it should only have the added Storybook story.